camel/agents/chat_agent.py

```diff
@@ -11,10 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
-from typing import Any, Dict, List, Optional, Tuple
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional
 
 import openai
-from tenacity import retry, stop_after_attempt, wait_exponential
+from tenacity import retry
+from tenacity.stop import stop_after_attempt
+from tenacity.wait import wait_exponential
```

Comment:
What is the benefit here?

Reply:
This was a warning from intellisence since stop_after_attempt and wait_exponential were in different subpackages